### DISCUSSION TITLE
add support for `align-items: baseline`

### DISCUSSION THREAD

**Discussion by stefanosambruna** [buildo/bento-design-system#427] [Ideas]
Add support for `align-items: baseline` to all the layout components that support this property. 

<img width="432" alt="Screenshot 2022-11-04 at 17 54 50" src="https://user-images.githubusercontent.com/24191264/200032146-168821f7-fd85-4f95-a64a-4a7b3c5cda86.png">

Why do I think it's important?
I would like to have on the same line two texts with different sizes (`line-height`, `font-size` etc.). For instance:
```
<Inline space={24} alignY="bottom">
   <Display size="medium">OOOOO</Display>
   <Headline size="medium">OOOOO</Headline>
</Inline>
```
which, I guess, corresponds to `align-items: bottom` and it results in:

<img width="735" alt="Screenshot 2022-11-04 at 17 51 03" src="https://user-images.githubusercontent.com/24191264/200031503-dc11c83e-3ed8-4306-b4c5-478c4e41acae.png">

As you can see from the image above the two texts do not look aligned. If I use `baseline` instead, I get something like this:

<img width="729" alt="Screenshot 2022-11-04 at 17 53 19" src="https://user-images.githubusercontent.com/24191264/200031990-46f3579c-50e3-4b1a-92d1-0912ecc13241.png">

**Comment by gabro** [comment]
Done in #446. It will be released in 0.15.2
